lua/oil/init.lua

```diff
@@ -273,6 +273,7 @@ M.open_float = function(dir)
     border = config.float.border,
     zindex = 45,
   }
+  win_opts = config.float.override(win_opts)
```

Comment:
With an API like this it can be difficult for users to notice or remember if the customization is in-place or a return value, so it's nice to support both. If they modify it in place and forget to return the config, we can support that by adding a `or win_opts` to the end of this line.